iib/workers/tasks/build.py

```diff
@@ -807,6 +814,11 @@ def handle_add_request(
             'index.Dockerfile',
         )
 
+        if is_fbc:
```

Comment:
Wouldn't `deprecatetruncate` change for fbc?

Reply:
Thanks, this totally slipped my mind, I added raising exception because `deprecatetruncate` will be implemented in different Jira.

Reply:
Just to clarify: I added exception when we want deprecate_bundles from FBC.Because it will be done in https://github.com/release-engineering/iib/pull/340